tests/test_tokenization.py

```diff
@@ -0,0 +1,140 @@
+"""Tests for GECToR custom tokenization"""
+import sys
+import os
+
+SCRIPT_DIR = os.path.dirname(os.path.abspath(__file__))
+sys.path.append(os.path.dirname(SCRIPT_DIR))
+
+from gector.tokenization import *
+import unittest
+from transformers import AutoTokenizer
+
+class TokenizationTests(unittest.TestCase):
+    """A test class that tests for the GECToR custom tokenization."""
```

Comment:
I guess in hindsight that tokenization.py doesn't really contain a tokenizer but rather a group of utility functions for manipulating the output of an existing BPE/wordpiece tokenizer.
So maybe we should say that it tests the GECToR custom tokenization utilities?